web/css/source/_theme.less

```diff
@@ -381,9 +393,11 @@ h1.site-intro{
 
 //Checkout page tweaks
 @media screen and (min-width: 768px){
```

Comment:
I wonder if the actual mistake here was using `min-width` instead of `max-width`?  Having them stacked seems right for mobile devices.  *shrug*